src/resolvers/AstExplorerResolver.ts

```diff
@@ -0,0 +1,84 @@
+import { readFile, writeFile } from 'mz/fs';
+import { URL } from 'whatwg-url';
+import NetworkResolver from './NetworkResolver';
+
+const EDITOR_HASH_PATTERN = /^#\/gist\/(\w+)(?:\/(\w+))?$/;
```

Comment:
I just changed this to not include the hostname, since we're supposed to be able to customize it in the constructor argument.